modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -9,13 +9,42 @@ module ClaimsApi
   module V2
     module Veterans
       class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
+        before_action :set_lighthouse_claim
+        before_action :set_bgs_claim!
+        before_action :validate_sponsor!
+        before_action :validate_file_number!
+        before_action :verify_if_dependent_claim!
```

Comment:
We should remove these from 5103... it should be running off of the participant ID with this change, which means
1. The File Number check is no longer relevant, we'll always pull a PID from the returned BGS call to use in the upload.
2. We're no longer worried about the dependent claim, that check was to assure we were receiving a sponsorIcn for any claims that matched a category. If the vet_id and claimant_id differ in the BGS request, we should assume it's of type dependent claim. BGS should be doing the "heavy lifting" there.

Similar to the removal of the sponsorIcn: https://github.com/department-of-veterans-affairs/vets-api/commit/72526d7fbc827f82bb80a88da0234ef130bf6edd